backend/inspirehep/submissions/views.py

```diff
@@ -259,12 +260,23 @@ def create_ticket(self, record, rt_template):
         )
 
 
+class ExperimentSubmissionsResource(BaseSubmissionsResource):
+    def post(self):
+        """Adds new experiment record"""
+
+        data = self.load_data_from_request()["data"]
+        data["_collections"] = ["Experiments"]
+        data["$schema"] = "https://labs.inspirehep.net/schemas/records/experiments.json"
+        record = ExperimentsRecord(data=data).create(data)
+        db.session.commit()
+        return (record, 201)
```

Comment:
are you sure we want to return a tuple of record and int 201? :p 
I think that you want to return `control_number` and status code using `jsonify`